modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -50,10 +52,104 @@ For a comprehensive list of supported dynamic plugins, see the link:{LinkPlugins
 
 With this update, theme configurations are enhanced to change the look and feel of different UI components so that they almost resemble the theme that is usually used in designing {company-name} applications. You might notice changes in UI components, such as buttons, tabs, sidebars, cards, and tables along with some changes in background color and font used on the {product-very-short} pages.
 
+You can update the `app-config.yaml` file to change the look of multiple {product-short} theme components for enhanced customization.
+
 == `scaffolderFieldExtensions` configuration option
 
 You can now use the `scaffolderFieldExtensions` configuration option in a dynamic plugin's front-end configuration. The `scaffolderFieldExtensions` option allows a dynamic plugin to specify one or more exported components to be provided to the scaffolder plugin as field extensions. These scaffolder field extensions provide custom form field components for the software template wizard.
 
 == Enhancement to ConfigMap or Secret configuration
 
-In previous versions, updating ConfigMaps or Secrets specified in `Backstage.spec.Application` required recreating the Pod to apply changes. Beginning with version {product-version}, this process is automated.
\ No newline at end of file
+In previous versions, updating ConfigMaps or Secrets specified in `Backstage.spec.Application` required recreating the Pod to apply changes. Beginning with version {product-version}, this process is automated.
+
+== Ability to configure learning paths
+
+You can now configure Learning Paths in {product-short} to create a dynamic experience tailored to your specific learning needs.
+
+== Plugin version upgrades in {product} 1.2.2
```

Comment:
@nickboldt Are there any NEW plugins introduced in 1.2.2?

Reply:
No, because this is a z-stream maintenance update, not a y-stream release with new features. 1.3 has currently one new plugin.